js/contextMenu.js

```diff
@@ -37,32 +18,37 @@ var template = [
 				label: 'Reload',
 				accelerator: 'CmdOrCtrl+R',
 				click: function(item, focusedWindow) {
-					if (focusedWindow)
+					if (focusedWindow) {
```

Comment:
oh... braces are optional for 1 line statements, forgot that. Glad you added them though.